### PR TITLE
refactor: license inconsistencies #2309 

### PR DESCRIPTION
**Pull request by bhavyagor12**
### Pull Request Checklist

- [ ] I have linked this PR to a related GitHub issue.
- [ ] I have added a description of the change (and Github issue number, if any) to the root [CHANGELOG.md](https://github.com/TryQuiet/quiet/blob/develop/CHANGELOG.md).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -262,3 +262,6 @@
 * C4 for Quiet architecture. Context and Container diagrams.
 
 * Invite tab as default in settings
+
+[2.0.0-alpha.12]
+* Refactored package.json to have consistent license "GPL-3.0-or-later"
```
Comment (leblowl) [inline CHANGELOG.md]: Would you mind moving this up to the top under unreleased?

**Review by leblowl — CHANGES_REQUESTED** [review]
Thanks, just one small comment.

File: CHANGELOG.md (hunk 1 above)
Comment (bhavyagor12) [reply CHANGELOG.md]: @leblowl could you pls check now? does this work

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,8 @@
 [unreleased]
 
+[2.0.0-alpha.12]
```
Comment (leblowl) [inline CHANGELOG.md]: I'd remove this line, sorry for the confusion

File: CHANGELOG.md (hunk 1 above)
Comment (leblowl) [reply CHANGELOG.md]: @bhavyagor12 one more small suggestion

File: CHANGELOG.md (hunk 2 above)
Comment (bhavyagor12) [reply CHANGELOG.md]: cool removing, no worries!

Comment (bhavyagor12) [reply CHANGELOG.md]: done @leblowl

**Review by leblowl — APPROVED** [review]

**Comment by leblowl** [comment]
Thanks!

**Pull request merged**